core/block.js

```diff
@@ -1673,7 +1674,8 @@ Blockly.Block.prototype.interpolate_ = function(message, args, lastDummyAlign) {
           input.setCheck(element['check']);
         }
         if (element['align']) {
-          input.setAlign(alignmentLookup[element['align']]);
+          var alignment = !!element['align'] && element['align'].toUpperCase();
+          input.setAlign(alignmentLookup[alignment]);
```

Comment:
If that alignment doesn't exist this will set the input's alignment to undefined. Should this throw if the lookup doesn't exist?

Reply:
I propose we default to Blockly.ALIGN_LEFT in that case and warn.

Reply:
SGTM.